pkgs/development/python-modules/python-bip32/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pytestCheckHook
+, pythonOlder
+, base58
+, coincurve
+}:
+
+buildPythonPackage rec {
+  pname = "python-bip32";
+  version = "3.4";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
Could you please use `pyproject = true` instead?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters